apps/sveltekit-example-app/package.json

```diff
@@ -20,14 +20,15 @@
     "pg": "8.12.0"
   },
   "devDependencies": {
-    "@packages/auth-lucia": "workspace:*",
+    "@packages/api": "workspace:*",
```

Comment:
Moving forward the only package that should be bought in should is @packages/api